critiquebrainz/frontend/views/review.py

```diff
@@ -211,8 +214,10 @@ def create():
             return redirect(url_for('user.reviews', user_id=current_user.id))
 
         is_draft = form.state.data == 'draft'
+        if not form.text.data:
```

Comment:
Can you make a comment noting this? Or, even better, just compare to an empty string.